From 97343d569751f094db91a5488eb8faec4eba7996 Mon Sep 17 00:00:00 2001
From: Jason Liu <jason.hui.liu@nxp.com>
Date: Wed, 25 Jul 2018 18:23:47 +0800
Subject: [PATCH 4300/5242] MLK-19057 soc: imx8qm: Enable
 TKT340553_SW_WORKAROUND on i.MX8QM RevB

commit  bf6620f71212de66645139c1fa2dbcbbae047043 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enabled the TKT340553_SW_WORKAROUND on i.MX8QM RevB

Signed-off-by: Jason Liu <jason.hui.liu@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
(cherry picked from commit 35cfd2953a795caa93312394a05c4811c9d3bd6c)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/imx/soc-imx8.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/imx/soc-imx8.c b/drivers/soc/imx/soc-imx8.c
index 279ac5a..2b28269 100644
--- a/drivers/soc/imx/soc-imx8.c
+++ b/drivers/soc/imx/soc-imx8.c
@@ -148,7 +148,7 @@ static u32 imx8qm_soc_revision(void)
 {
 	u32 rev = imx_init_revision_from_scu();
 
-	if (rev == IMX_CHIP_REVISION_1_0)
+	if (rev == IMX_CHIP_REVISION_1_0 || rev == IMX_CHIP_REVISION_1_1)
 		TKT340553_SW_WORKAROUND = true;
 
 	return rev;
-- 
1.7.9.5

